src/overlays/actors/ovl_Dm_Sa/z_dm_sa.h

```diff
@@ -5,9 +5,19 @@
 
 struct DmSa;
 
+typedef void (*DmSaActionFunc)(struct DmSa*, GlobalContext*);
+
 typedef struct DmSa {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x1B0];
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ char unk194[0x108];
```

Comment:
I generally like to keep these unk fields UNK_TYPE1 if it still matches, to signal to the reader of this code that we don't know the signedness of this data.